src/models/token.model.js

```diff
@@ -0,0 +1,22 @@
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db');
+const { User } = require('./user.model');
+
+const Token = sequelize.define(
+  'token',
+  {
+    refreshToken: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    createdAt: false,
+    updatedAt: false,
+  },
+);
+
+Token.belongsTo(User);
+User.hasOne(Token);
```

Comment:
Consider using `User.hasMany(Token)` if users can have multiple tokens. This would better represent the relationship if multiple tokens per user are allowed.